src/foam/nanos/pm/PMInfo.js

```diff
@@ -4,54 +4,53 @@
  * http://www.apache.org/licenses/LICENSE-2.0
  */
 
-// TODO: rename properties to use camelCase
 foam.CLASS({
   package: 'foam.nanos.pm',
   name: 'PMInfo',
 
   documentation: 'Performance Measurement database entry.',
 
-  ids: [ 'clsname', 'pmname' ],
+  ids: [ 'clsName', 'pmName' ],
 
   searchColumns: [ ],
 
   properties: [
     {
       class: 'String',
-      name: 'clsname',
+      name: 'clsName',
       label: 'Class'
     },
     {
       class: 'String',
-      name: 'pmname',
+      name: 'pmName',
       label: 'Name'
     },
     {
       class: 'Int',
-      name: 'numoccurrences',
+      name: 'numOccurrences',
```

Comment:
I think just 'count' would be better. Who can remember how to spell occurrences.

Reply:
Agree, changing it now.